Automattic/jetpack
# CRM: Resolves 3106 - Emerald learn menu

Resolves Automattic/zero-bs-crm#3106 - Emerald learn menu

## Proposed changes:

This implements the Emerald learn menu (under the top menu and above the main content).
Old:
![image](https://github.com/Automattic/jetpack/assets/32492176/bde4d469-a9c6-4362-836c-e06e242afa1b)

New:
![image](https://github.com/Automattic/jetpack/assets/32492176/8930dee1-dc92-4daf-b829-c2d37674286e)


There is a new `jpcrm_learn_menus` filter that can be used like so:
```
function jpcrm_some_extension_learn_menu( $learn_menu_array ) {

	$learn_menu_array[ 'mail-assistant' ] = array(
		'title'         => 'My Extension',
		'url'           => `https://my_url.dev.null`,
		'img'           => '',
		'video'         => false,
		'content'       => '<p></p>',
	);
	return $learn_menu_array;
}
add_filter( 'jpcrm_learn_menus', 'jpcrm_some_extension_learn_menu' );
```

Known issues:
* Extensions may not look pretty (here's looking at you, Sales Dashboard). That said, they should still work.
* The learn menu popover overlays the admin bar if one scrolls up with it open.
* Small screens are out of scope.
* Various oddities too numerous to list here do exist. Feel free to comment if you run across them.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Click through any page you can. Of note:

* Page titles in the learn menu should be correct
* Listviews should always have "Add object" and "Table options" buttons
* Dashboard and Contact new/edit pages should have a "Page Options" button
* Next/previous navs should work if enabled on contacts/companies
* CSV Importer Lite should properly show the menu throughout its steps
